doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -539,6 +539,112 @@ Requesting information
    fixpoint and cofixpoint is violated at some time of the construction
    of the proof without having to wait the completion of the proof.
 
+.. _showing_diffs:
+
+Showing differences between proof steps
+---------------------------------------
+
+
+Coq can automatically highlight the differences between successive proof steps.
+For example, the following screen shots of coqtop and CoqIDE show the same
+“intros” tactic.  This tactic creates two new hypotheses, highlighted in green.
+The conclusion is entirely in pale green because it’s changed, but no tokens were added
+to it.  The third screen shot uses the "removed" option, so it shows the conclusion a
+second time with the old text, with deletions marked in red.  Also, since the hypotheses are
+new, no line of old text is shown for them.
+
+.. _diffs_coqtop_on:
+
+  .. image:: ../_static/diffs-coqtop-on.png
```

Comment:
I would crop the top of this image which is confusing.

Reply:
I thought I should show the prior state state (the `forall` line) for context.  I'll include the `forall` as regular text.

Reply:
I think what `intros` does is sufficiently clear to everyone that you don't need to give the context.